installer/charts/values.yaml.tpl

```diff
@@ -6,9 +6,12 @@
 {{- $gitops := required "GitOps settings" .Installer.Features.openShiftGitOps -}}
 {{- $pipelines := required "Pipelines settings" .Installer.Features.openShiftPipelines -}}
 {{- $quay := required "Quay settings" .Installer.Features.redHatQuay -}}
+{{- $nexusSecret := (lookup "v1" "Secret" .Installer.Namespace "rhtap-nexus-integration") -}}
+{{- $quaySecret := (lookup "v1" "Secret" .Installer.Namespace "rhtap-quay-integration") -}}
+{{- $quayEnabled := not (or $nexusSecret $quaySecret) -}}
 {{- $rhdh := required "RHDH settings" .Installer.Features.redHatDeveloperHub -}}
 {{- $ingressDomain := required "OpenShift ingress domain" .OpenShift.Ingress.Domain -}}
-{{- $minIOOperatorEnabled := or $tpa.Enabled $quay.Enabled -}}
+{{- $minIOOperatorEnabled := or $tpa.Enabled $quayEnabled -}}
```

Comment:
That would install minio if there was Nexus and no TPA. Feels wrong. I don't think you need to change that condition, here or later in the file.